src/Arcus.WebApi.Security/Authorization/AzureManagedIdentityAuthorizationFilter.cs

```diff
@@ -1,35 +0,0 @@
-﻿using System.Threading.Tasks;
-using Arcus.WebApi.Security.Authorization.Jwt;
-using GuardNet;
-using Microsoft.AspNetCore.Mvc;
-using Microsoft.AspNetCore.Mvc.Filters;
-
-namespace Arcus.WebApi.Security.Authorization
-{
-    public class AzureManagedIdentityAuthorizationFilter : IAsyncAuthorizationFilter
```

Comment:
We have already shipped it so we'll have to deprecate it and just inherit from the new one

Reply:
Ah, yes of course. Sorry about that 😞 , Wasn't thinking.

Reply:
No problem at all!